game-core/build.gradle

```diff
@@ -16,7 +14,7 @@ dependencies {
     implementation "org.apache.commons:commons-math3:$commonsMathVersion"
     implementation "org.apache.httpcomponents:httpclient:$apacheHttpComponentsVersion"
     implementation "org.apache.httpcomponents:httpmime:$apacheHttpComponentsVersion"
-    implementation "org.mindrot:jbcrypt:$jbcryptVersion"
+    implementation "at.favre.lib:bcrypt:$bcryptVersion"
```

Comment:
I don't see any other changes in the `game-core` project.  Is this dependency still needed?

Reply:
Apparently not.
Good catch